vendor/parse5/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
Ideally, none of the parse5 files below need to be checked in now that we're using browserify.